test/run

```diff
@@ -17,6 +17,13 @@
 
 THISDIR=$(dirname ${BASH_SOURCE[0]})
 
+# Some tests, like the one using the latest pipenv, might be unstable
+# because new upstream releases tend to break our tests sometimes.
+# If a test is in UNSTABLE_TESTS and IGNORE_UNSTABLE_TESTS env
+# variable is defined, a result of the test has no impact on
+# the overall result of the test suite.
+declare -a UNSTABLE_TESTS=(pino)
```

Comment:
Please note, that this will not work, unless the container tests are run with the `ct_run_tests_from_testset` function.

Reply:
ok, let's do it in separate PR. I will remove it.